### ISSUE TITLE
Deprecation Warning for Distance Utility

### ISSUE THREAD

**Issue by ChrisRomp** [jugla/worldtidesinfocustom#20]
**Describe the bug**
HA is reporting that the distance utility is deprecated since 2022.10 and will stop working in 2023.4. It should be updated to use `unit_conversion.DistanceConverter` instead.

**To Reproduce**
Use component with HA version > 2022.10 and view the `home-assistant.log` file.

**Expected behavior**
No deprecation warning

**Additional context**
Log output:
```
2022-12-27 09:16:04.806 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that uses distance utility. This is deprecated since 2022.10 and will stop working in Home Assistant 2023.4, it should be updated to use unit_conversion.DistanceConverter instead. Please report issue to the custom integration author for worldtidesinfocustom using this method at custom_components/worldtidesinfocustom/live_position_management.py, line 18: KM_PER_MI = dist_convert(1, LENGTH_MILES, LENGTH_KILOMETERS)
2022-12-27 09:16:04.807 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that uses distance utility. This is deprecated since 2022.10 and will stop working in Home Assistant 2023.4, it should be updated to use unit_conversion.DistanceConverter instead. Please report issue to the custom integration author for worldtidesinfocustom using this method at custom_components/worldtidesinfocustom/live_position_management.py, line 19: MI_PER_KM = dist_convert(1, LENGTH_KILOMETERS, LENGTH_MILES)
2022-12-27 09:16:04.808 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that uses distance utility. This is deprecated since 2022.10 and will stop working in Home Assistant 2023.4, it should be updated to use unit_conversion.DistanceConverter instead. Please report issue to the custom integration author for worldtidesinfocustom using this method at custom_components/worldtidesinfocustom/live_position_management.py, line 20: FT_PER_M = dist_convert(1, LENGTH_METERS, LENGTH_FEET)
2022-12-27 09:16:04.811 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that uses distance utility. This is deprecated since 2022.10 and will stop working in Home Assistant 2023.4, it should be updated to use unit_conversion.DistanceConverter instead. Please report issue to the custom integration author for worldtidesinfocustom using this method at custom_components/worldtidesinfocustom/worldtides_data_coordinator.py, line 20: KM_PER_MI = dist_convert(1, LENGTH_MILES, LENGTH_KILOMETERS)
2022-12-27 09:16:04.812 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that uses distance utility. This is deprecated since 2022.10 and will stop working in Home Assistant 2023.4, it should be updated to use unit_conversion.DistanceConverter instead. Please report issue to the custom integration author for worldtidesinfocustom using this method at custom_components/worldtidesinfocustom/worldtides_data_coordinator.py, line 21: MI_PER_KM = dist_convert(1, LENGTH_KILOMETERS, LENGTH_MILES)
2022-12-27 09:16:04.814 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that uses distance utility. This is deprecated since 2022.10 and will stop working in Home Assistant 2023.4, it should be updated to use unit_conversion.DistanceConverter instead. Please report issue to the custom integration author for worldtidesinfocustom using this method at custom_components/worldtidesinfocustom/worldtides_data_coordinator.py, line 22: FT_PER_M = dist_convert(1, LENGTH_METERS, LENGTH_FEET)
```

**Comment by jugla** [comment]
Thanks !

**Comment by jugla** [comment]
I release v12.3.0.
Enjoy !

**Comment by ChrisRomp** [comment]
Fixed via 20fb8e658e1f19965a0bc7d8d0aacfcdfaf92832.
